applications/sofa/gui/headlessRecorder/HeadlessRecorder.cpp

```diff
@@ -346,7 +346,9 @@ int HeadlessRecorder::mainLoop()
         }
         else
         {
-            sleep(0.01);
+            //TODO(hugtalbot 2020-04-17): back to zero since sleep(0.01) creates implicit
+            //conversion from 'double' to 'unsigned int' changes value from 0.01 to 0
+            sleep(0);
```

Comment:
I don't see the point to sleep 0 second 😺 
and it would be better to use sofa::helper::system::thread::CTime::sleep(10) as it is system/OS independent (it takes double and it represents milliseconds)
Or just use std::this_thread::sleep_for(std::chrono::milliseconds(10));

Reply:
😄 obviously the 🧵  is ": thread :"